ExperimentManager1.py

```diff
@@ -255,7 +277,9 @@ def start_streaming_exp(self, num_trial, duration, time_decision, opponent_type,
         state_history = []
         while currentstate != States.End:
             # print(f"Current State: {currentstate}")
-            state_history.append(currentstate)
```

Comment:
need it as i might need the state history for the simulated learner

Reply:
I do not understand why you need it. Let's discuss. You have the same information in the experiment log file. Also, You do not need the array to be appended every cycle of the loop. Move it to state changes if you really need it.

Reply:
can you keep this line

Reply:
Its still there don't worry